tests/Auth/AuthGuardTest.php

```diff
@@ -521,6 +520,6 @@ protected function getMocks()
 
     protected function getCookieJar()
     {
-        return new CookieJar(Request::create('/foo', 'GET'), m::mock(Encrypter::class), ['domain' => 'foo.com', 'path' => '/', 'secure' => false, 'httpOnly' => false]);
+        return new CookieJar();
```

Comment:
Why did you remove the arguments here?

Reply:
:thinking: There wasn't any usage of them! I'm going to revert this change.